versions.tf

```diff
@@ -6,5 +6,10 @@ terraform {
       source  = "spacelift-io/spacelift"
       version = ">= 1.14"
     }
+
+    jsonschema = {
```

Comment:
I believe this requires running `terraform-docs`.

Reply:
I'll fix this in a follow up!